Tasks/AzureRmWebAppDeployment/AzureRMUtil.ts

```diff
@@ -0,0 +1,293 @@
+/// <reference path="../../definitions/node.d.ts" />
+/// <reference path="../../definitions/q.d.ts" />
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+
+var adal = require ('adal-node');
+
+import taskLib = require ('vsts-task-lib/task');
+var request = require ('request');
+import Q = require ('q');
+var parseString = require('xml2js').parseString;
+
+var AuthenticationContext = adal.AuthenticationContext;
+
+function getAuthorizationToken ( SPN ) : Q.Promise <string> {
```

Comment:
formatting doesn't look good - lots of spaces between tokens. is this the formatting set by some vscode config in root folder? if not, check other node tasks in this repo and correct your local settings, please.